Dockerfile

```diff
@@ -1,26 +1,31 @@
 ARG GOLANG_VERSION=1.19
-ARG MENDER_CLI_VERSION=master
-ARG MENDER_ARTIFACT_VERSION=master
-ARG MENDER_CLIENT_VERSION=master
+ARG MENDER_CLI_VERSION=1.9.0
+ARG MENDER_ARTIFACT_VERSION=3.9.0
+ARG MENDER_CLIENT_VERSION=3.4.0
 
 FROM golang:${GOLANG_VERSION} as cli-builder
 WORKDIR /go/src/github.com/mendersoftware/mender-cli
 ARG MENDER_CLI_VERSION
-RUN git clone -b $MENDER_CLI_VERSION https://github.com/mendersoftware/mender-cli.git . && \
+RUN git clone https://github.com/mendersoftware/mender-cli.git . && \
+    git checkout $MENDER_CLI_VERSION && \
     make get-deps && \
     make build
 
 FROM golang:${GOLANG_VERSION} as artifact-builder
 WORKDIR /go/src/github.com/mendersoftware/mender-artifact
 ARG MENDER_ARTIFACT_VERSION
-RUN git clone -b $MENDER_ARTIFACT_VERSION https://github.com/mendersoftware/mender-artifact.git . && \
-    make get-build-deps && \
+RUN git clone https://github.com/mendersoftware/mender-artifact.git . && \
+    git checkout $MENDER_ARTIFACT_VERSION && \
+    make get-build-deps || ( \
+        apt-get update -qq && \
+        apt-get install -yyq $(cat deb-requirements.txt) ) && \
```

Comment:
It looks like overkill until it's some specific reason for copy-paste from the Makefile here.

Reply:
Latest released `mender-artifact` does not have this target. I tried to justify it in the commit message, should I rephrase it somehow if it is not clear the reason?

Reply:
Sorry, didn't look to the commits.

But shouldn't 'make' have some other target for this purpose? If not then what is the reason of removing this useful target?

Reply:
The target is good, is just that it only lives in `master` as per today. Whenever we release new mender-artifact it will be released and then `mender-ci-workflows` repo can use it.